Translation-Italian.md

```diff
@@ -0,0 +1,175 @@
+[English](./readme.md) ·
+[中文](./Translation-Chinese.md) ·
+[Español](./Translation-Spanish.md) ·
+[Português](./Translation-Portuguese.md) ·
+[Français](./Translation-French.md) ·
+[Русский](./Translation-Russian.md) ·
+[Deutsch](./Translation-German.md) ·
+[日本語](./Translation-Japanese.md) ·
+Italiano 
+
+---
+
+# LaTeX in pochi minuti
+
+![](https://upload.wikimedia.org/wikipedia/commons/9/92/LaTeX_logo.svg)
+
+**Riconoscimenti:** *Quanto scritto in questo documento deriva da mie esperienze universitarie e letture di diverso materiale sull'argomento. Non sono un professionista o un esperto ma un semplice studente con una passione per l'argomento. Chiunque può aprire una discussione nella sezione "Problemi" o aprire una Pull Request nel caso ritenga che qualcosa debba essere aggiunto o modificato. Se il mio lavoro vi è tornato utile una [donazione](#donation) è molto apprezzata.*
+
+### Sommario
+
+* [Che cos'è LaTeX?](#che-cosè-latex)
```

Comment:
Yeah this is a tricky thing to handle. A nice thing to know is that apostrophe = %27, but Javascript is notorious for ignoring this character in encoding the URL, FYI. Source: https://www.google.com/search?client=firefox-b-d&q=encodeURIComponent+didn%27t+encode+single+quote.  This is why the original  ' doesnt work